### ISSUE TITLE
d2l-table causes scrollable-region-focusable accessibility violation

### ISSUE THREAD

**Issue by Vitalii-Misechko** [BrightspaceUI/core#1363]
```
 Rule: scrollable-region-focusable
      Impact: moderate
      Ensure that scrollable region has keyboard access (https://dequeuniversity.com/rules/axe/4.2/scrollable-region-focusable?application=axeAPI)

      Issue target: d2l-insights-assessment-quality-dashboard,d2l-table-wrapper,d2l-scroll-wrapper,.d2l-scroll-wrapper-container
      Context: <div class="d2l-scroll-wrapper-container"><slot></slot></div>
      Fix any of the following:
        Element should have focusable content
        Element should be focusable
```

Steps to reproduce:
1) Run `d2l-insights-engagement-dashboard.accessibility` test in [this repo](https://github.com/Brightspace/insights-assessment-quality-dashboard). Comment out `scrollable-region-focusable` inside `ignoredRules` block
